library/patient_tracker.inc.php

```diff
@@ -32,9 +32,12 @@
 use OpenEMR\Services\AppointmentService;
 use OpenEMR\Services\PatientTrackerService;
 
+$apppointmentService = new AppointmentService();
+$patientTrackerService = new PatientTrackerService();
```

Comment:
Are their specific queries/code that are slow in these classes during their initiation?

Reply:
Not seeing slow instantiation, bogs down from hundreds of unnecessary new objects.